lib/internal/cli_table.js

```diff
@@ -2,7 +2,7 @@
 
 const { Buffer } = require('buffer');
 const { removeColors } = require('internal/util');
-const HasOwnProperty = Function.call.bind(Object.prototype.hasOwnProperty);
+const hasOwnProperty = Function.call.bind(Object.prototype.hasOwnProperty);
```

Comment:
I know @bmeck has also been doing it this way, (at least guessing from the reasons that I use it) that it matches the name the spec uses?

Reply:
@BridgeAR are you attached to this particular change? It's the only thing stopping me from approving & landing to be honest and I've looked at this PR several times. I believe we should have a convention and follow it — at the moment uppercase is the way we do this in most files. I'm not opposed to switching but I don't think it should be done incrementally.

Reply:
I switched it back even though this is really weird for me as it is not a class. IMO we should only have upper case characters for classes.